Tema OOP/src/main/java/MainTema.java

```diff
@@ -0,0 +1,59 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainTema {
+    public static void main(String[] args) {
+        Book c1 = new Book();
+        c1.setNume("Prima Carte");
+        c1.setNumarPagini(30);
+
+        Book c2 = new Book();
+        c2.setNume("A doua Carte");
+        c2.setNumarPagini(80);
+
+        Book c3 = new Book();
+        c3.setNume("A treia Carte");
+        c3.setNumarPagini(550);
+
+        Novel n1 = new Novel();
+        n1.setNume("Prima Nuvela");
+        n1.setNumarPagini(200);
+        n1.setType("Mystery");
+
+        Novel n2 = new Novel();
+        n2.setNume("A doua Nuvela");
+        n2.setNumarPagini(350);
+        n2.setType("Science fiction");
+
+        ArtAlbum a1 = new ArtAlbum();
+        a1.setNume("Primul Album");
+        a1.setNumarPagini(35);
+        a1.setCaliateHartie("Buna");
+
+        ArtAlbum a2 = new ArtAlbum();
+        a2.setNume("Al doilea Album");
+        a2.setNumarPagini(55);
+        a2.setCaliateHartie("Foarte buna");
+
+        List<Book> listaCarti = new ArrayList<>();
```

Comment:
no need for this.
please create :
Library aLibrary = new Library();
aLibrary.setListOfBooks(listaCarti);
aLibrary.add..
aLibrary.remove...
aLibrary.list();..